input/elasticapm/internal/modeldecoder/v2/decoder.go

```diff
@@ -1329,6 +1366,11 @@ func mapToTransactionModel(from *transaction, event *model.APMEvent) {
 	}
 }
 
+func mapToTelemetryModel(from *telemetry, event *model.APMEvent) {
+	event.Processor = model.TelemetryProcessor
+	// TODO : finish mapping
+}
```

Comment:
As we might not map this data to an actual persisted model, this might be a good place to send the data directly to the telemetry endpoint if we can do that directly from apm-data.